README.md

```diff
@@ -8,7 +8,7 @@ This is a [fonk](https://github.com/Lemoncode/fonk) microlibrary that brings val
 
```

Comment:
Add more info about BIC code, short description, plus link to wikipedia: https://es.wikipedia.org/wiki/ISO_9362

Reply:
More info (metion the source: : https://www.theswiftcodes.com)

Swift Code is a standard format of Bank Identifier Codes (BIC) and it is unique identification code for a particular bank. These codes are used when transferring money between banks, particularly for international wire transfers. Banks also used the codes for exchanging other messages between them.
The Swift code consists of 8 or 11 characters. When 8-digits code is given, it refers to the primary office. The code formatted as below;

AAAA BB CC DDD

First 4 characters - bank code (only letters)
Next 2 characters - ISO 3166-1 alpha-2 country code (only letters)
Next 2 characters - location code (letters and digits) (passive participant will have "1" in the second character)
Last 3 characters - branch code, optional ('XXX' for primary office) (letters and digits)

Reply:
Ok, done